libcontainer/standard_init_linux.go

```diff
@@ -79,6 +79,14 @@ func (l *linuxStandardInit) Init() error {
 	}
 
 	label.Init()
+
+	// Use Unshare() to enter a new cgroup_namespace before InitializeMountNamespace()
+	if l.config.Config.Namespaces.Contains(configs.NEWCGROUP) && l.config.Config.Namespaces.PathOf(configs.NEWCGROUP) == "" {
+		if err := syscall.Unshare(configs.Syscall_CLONE_NEWCGROUP); err != nil {
```

Comment:
I'd be happy to give you a hand if you have any questions, I didn't mean to scare you off. :smile_cat: